Update MacOS installation information

pcolladosoto opened the pull request:
**Detailed description**
We have included information on how to install Rizin both through Homebrew and MacPorts on top of using the package file provided on the releases.

**Closing issues**
This PR is intended to add information on the installation procedures offered after closing issue #160.

pcolladosoto committed:
Update MacOS installation information

We have included information on how to install
Rizin both through Homebrew and MacPorts on
top of using the package file provided on
the releases.

ret2libc commented on INSTALL.md:
```diff
@@ -30,15 +30,26 @@ anywhere because Rizin is compiled in a "portable" way, allowing moving the
+On MacOS systems you can install Rizin in one of three ways.
+
+### Homebrew
+Rizin offers a Homebrew [formula](https://formulae.brew.sh/formula/rizin)
```
both here and in the next section i feel we should be more explicit on the fact that it is not us who provide these things, but they are maintained by the respective communities (homebrew and macports). We might help them keep them updated or fix things here and there, but we are not actively keeping them updated.

pcolladosoto replied:
I added some information tackling this topic. What do you think?

wargio replied:
LGTM

ret2libc reviewed (commented):
Thanks a lot!

pcolladosoto committed:
Clarify policies of Homebrew and MacPorts

@ret2libc offered some advice on how it might be
positive to make it clear that Homebrew and
MacPorts maintain their own versions of Rizin.
This implies the version they offer might be a
bit outdated at some point...

ret2libc commented:
Thanks! ;)
